### DISCUSSION TITLE
application not accepted

### DISCUSSION THREAD

**Discussion by brunommorales** [community/community#51980] [GitHub Education]
I've tried in every way, I showed all the documents and even translated it into English and still all the answers were denied

**Comment by samus-aran** [comment]
Hi @brunommorales  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by brunommorales** [reply]
Yes, I can show you what happened. However, I sent all dated documents and also proof of college enrollment and semester classes.
What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.


You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
